gundi_core/schemas/v2.py

```diff
@@ -369,6 +369,79 @@ class IntegrationActionSummery(BaseModel):
         description="Short text id for the action, to be used programmatically",
     )
 
+class IntegrationWebhook(BaseModel):
+    id: Union[UUID, str] = Field(
+        None,
+        title="Integration Webhook ID",
+        description="Id of an webhook in Gundi",
+    )
+    name: Optional[str] = Field(
+        "",
+        example="Generic Webhook",
+        description="A human-readable name for the webhook",
+    )
+    value: Optional[str] = Field(
```

Comment:
Do you think it's worth applying validation on `value`? Maybe to limit case, string length, special characters?

Reply:
Yes, good idea. I should add those validations in the Gundi API too.